exampleoc/ocpath.go

```diff
@@ -97,6 +99,7 @@ func (n *AclPathAny) AclSetAnyType(Name string) *Acl_AclSetPathAny {
 }
 
 // AclSetAnyName returns from AclPath the path struct for its child "acl-set".
+// Type: E_Acl_ACL_TYPE
```

Comment:
I see examples in other files, where the type name is preceded by the "[modulename.] in front. Is that what. we can expect to happen in practice?

Reply:
In this case, as well as likely in practice, the path library is **the same** as the goStructs library, so there is no "modulename." prefix.

Reply:
Should it have the name of it's own self in front (the likely name by which it's imported), or just assume the user knows that?

Reply:
It's not possible to know what the user's import prefix is -- the user might give an alias to the package path, and then it's no longer the package name.